src/buza/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+from .models import BuzaUser
+
+
+class BuzaAdmin(admin.ModelAdmin):
+    # Register your models here.
+    list_display = ['user_name', 'school', 'reputation']
+
+
+admin.site.register(BuzaUser, BuzaAdmin)
```

Comment:
We can use the newer `@admin.register` decorator for admin registrations. :)

Docs: [The Django admin site: The register decorator](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/#the-register-decorator)